spec/api/appointments_spec.rb

```diff
@@ -4,7 +4,7 @@
   path '/appointments/sync' do
 
     post 'Syncs appointment data from device to server.' do
-      tags 'Follow Up Schedule'
+      tags 'Appointment'
```

Comment:
Why is this in this PR?

Reply:
Missed renaming these tags in the PR adding `Appointment` and `Communication`. 

Wanted to fix these in the PR itself, instead of raising another one, since it's a small change.

Reply:
If you don't mind, can you add those to another PR? If it's small it should be easy to review & merge.

Putting unrelated changes in can make future reference confusing.